Makefile

```diff
@@ -36,7 +36,7 @@ services: python
 .PHONY: db
 db: args?=upgrade head
 db: python
-	@tox -qe dev --run-command 'python bin/make_db'
+	@tox -qe dev --run-command 'python3 -m h.scripts.init_db --create --stamp'
```

Comment:
For context, `make db` is a convenience command that was added for non-experts (frontend developers) who didn't want to have to faff with Alembic and upgrading their DB schema. The idea is that if you're having a problem you can just run `make db` and it'll initialize your DB (in case it isn't already) and upgrade you to the latest Alembic version.

None of the other commands depend on `make db` and you don't have to use it if you don't want to.

The `$(args)` means that you can use `make db` as a shortcut for running Alembic, e.g. `make db args="upgrade head"` but you can also just run Alembic directly if you want to (`tox -e dev --run-command 'alembic upgrade head'`). (The `ALEMBIC_CONFIG` envvar in `tox.ini` tells Alembic about the `conf/alembic.ini` file so you don't have to do `--config conf/alembic.ini` all the time.)